aframe/utils.py

```diff
@@ -118,6 +118,8 @@ def get_num_shifts_from_num_injections(
     of background segments, injection spacing, and the desired total
     number of injections
     """
+    buffer += waveform_duration
```

Comment:
In `calc_segment_injection_times`, we do `buffer += waveform_duration // 2`. Highly unlikely to make a difference, but I think it would be good to match